tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -138,12 +158,21 @@ Users can see the pod table in each resource details page.
 
 ### <a id="pod-details-metrics"></a> Overview of pod metrics
 
-The overview card displays the user-configured resource limits on the pod, defined in accordance with
-the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
-These limits do not represent actual real-time resource use. To monitor actual real-time resource use,
-see [Application Live View for Spring Boot Applications in Tanzu Application Platform GUI](app-live-view-springboot.md).
+If you have a metric server running on your cluster, you will find that the
+overview card displays realtime metrics of your pod presented as a percentage
+of the max capability of this one.
+
+In the scenario where you have no metric server, the overview card displays the
+user-configured resource limits on the pod, defined in accordance with the
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
```

Comment:
Suggestion:

If you do not have a metrics server running on your cluster, the overview card will display the configured resource limits for pods.

(probably ok to omit the k8s docs part, because I think that applies to most things we present)

Reply:
Addressed, please consider resolved.